modules/web-discovery-project/sources/web-discovery-project.es

```diff
@@ -3965,7 +3962,7 @@ const WebDiscoveryProject = {
       var page_struct_before = page_doc["x"];
       url_pagedocPair[url] = page_doc;
 
-      WebDiscoveryProject.log(
+      logger.debug(
```

Comment:
I think there is another place to update here: `platforms/webextension/web-discovery-project/storage.es`